packages/react-dom/src/__tests__/ReactServerRenderingHydration.js

```diff
@@ -258,6 +258,45 @@ describe('ReactDOMServerHydration', () => {
     expect(element.firstChild.focus).not.toHaveBeenCalled();
   });
 
+  it('should warn when the style property differs', () => {
+    const element = document.createElement('div');
+    element.innerHTML = ReactDOMServer.renderToString(
+      <div style={{textDecoration: 'none', color: 'black'}} />,
+    );
+    expect(element.firstChild.style.textDecoration).toBe('none');
+    expect(element.firstChild.style.color).toBe('black');
+
+    expect(() =>
+      ReactDOM.hydrate(
+        <div style={{textDecoration: 'none', color: 'white'}} />,
+        element,
+      ),
+    ).toWarnDev(
+      'Warning: Prop `style` did not match. Server: ' +
+        '"text-decoration:none;color:black" Client: ' +
+        '"text-decoration:none;color:white"',
+      {withoutStack: true},
+    );
+  });
+
+  it('should not warn when the style property differs on whitespace only', () => {
+    const element = document.createElement('div');
+    element.innerHTML = ReactDOMServer.renderToString(
+      <div style={{textDecoration: 'none', color: 'black'}} />,
+    );
+    expect(element.firstChild.style.textDecoration).toBe('none');
+    expect(element.firstChild.style.color).toBe('black');
+
+    spyOnDevAndProd(console, 'error');
```

Comment:
You don't need this. We check that every test doesn't warn by default. So you can remove it.